src/nvim/api/ui.c

```diff
@@ -66,6 +66,11 @@ static void mpack_str_small(char **buf, const char *str, size_t len)
   *buf += len;
 }
 
+uint64_t get_ui_number(void)
```

Comment:
`ui_active` already gives this info https://github.com/neovim/neovim/blob/efd0fa55c8b1ba78692382c52a9d644fdd5488c8/src/nvim/ui.c#L186-L187